### DISCUSSION TITLE
Improve - Notifications

### DISCUSSION THREAD

**Discussion by davehakkens** [ONEARMY/community-platform#3231] [Big Updates]
This discussion is regarding our notifications. They are a bit patched together throughout the years and could use a clear overview and aligment between all the moving parts. 

The goal here is to identify the problems and discuss/share possible ways forward to see if it's worth putting on our roadmap.

# Current problems
|     Description            |        Image             |
 |----------------------------------------|------------------------------------------------------------|
 |     1. Messages are not descriptive enough([more info](https://github.com/ONEARMY/community-platform/issues/2922)) |<img width="236" alt="afbeelding" src="https://github.com/ONEARMY/community-platform/assets/13672737/f4e648ee-3215-4544-afc3-3b8d48115f8e"> |
 |     2. Popup window is small and not very intuitive to use (should you 'clear notifications' and what influence does this have on notification summary email)  | <img width="432" alt="afbeelding" src="https://github.com/ONEARMY/community-platform/assets/13672737/cefe77df-5c20-405c-9e10-7ea50bc7e412"> | 
 | 3. Nested comments are not included yet. They are still [being build](https://github.com/ONEARMY/community-platform/issues/2564) but notifications for this are essential to include in the plan | <img width="694" alt="afbeelding" src="https://github.com/ONEARMY/community-platform/assets/13672737/00495da1-9a51-4987-bba6-508a0f8f14cb">  |
| 4 Signup email+change password don’t receive styled email | <img width="640" alt="afbeelding" src="https://github.com/ONEARMY/community-platform/assets/13672737/ad5b0077-0de1-4304-a578-c97c4a3556d2"> |
| 5 Webhook notification in #activitybot in Discord need to be updated too include other modules and instances. (Here a specific thread for it)  | <img width="790" alt="afbeelding" src="https://github.com/ONEARMY/community-platform/assets/13672737/f58dfd1c-52eb-4e77-9baa-cec2ed83ee14"> |
| 6 Notifications triggers are not clear and documented for other developers in our [docs](https://docs.platform.onearmy.earth/) | <img width="790" alt="afbeelding" src="https://github.com/ONEARMY/community-platform/assets/13672737/f3f4222e-9c37-4848-9896-02e03c989694">
 |

# Things to keep in mind
- We have email + web notifications. Updates need to happen to both

# Whats needed next

- [ ] Create overview of all exiting notifications  (start made [here](https://github.com/ONEARMY/community-platform/issues/2922#issuecomment-1773929803))
- [ ] Make a plan on how to move forward with them 
- [ ] Document notifications in our [docs](https://docs.platform.onearmy.earth/)

**Comment by adricecilia** [comment]
Hey, my name is Adrian and I have seen that you have several points regarding notifications and I would like to know which ones are pending resolution to provide different points of view / ideas to solve them.

**Reply by davehakkens** [reply]
hey thanks for checking in and wanting to help!
This discussion here is a bit outdated as things developed.

Here is the latest issue for whats next for notifications: https://github.com/ONEARMY/community-platform/issues/4043
We are currently designing this so good time to share ideas or help out

**Comment by mariojsnunes** [comment]
It's been done as part of a recent effort
